lib/web/mage/backend/validation.js

```diff
@@ -33,6 +33,17 @@
                     $(this.errorList.length && this.errorList[0].element || [])
                         .focus()
                         .trigger('focusin');
+                    $('[data-toggle=dropdown].active').trigger('open.dropdown');
+                    let first = true;
```

Comment:
Magento support just ES5, `let` variables it is the feature of ES6, so currently we shouldn't use it. Please use `var` instead of `let` or `const`